packages/frontend/src/components/Task/Task.tsx

```diff
@@ -2,31 +2,46 @@ import { Button, Image, Input, Text } from '@mantine/core'
 import { BsPaperclip, MdMessage } from 'react-icons/all'
 import { MdAdd } from 'react-icons/md'
 import { useState } from 'react'
-import PropTypes from 'prop-types'
+import { RandomUserType } from '../../logic/randomUser'
 import useStyles from './style'
 import UserIconList from '../UserIconList/UserIconList'
 
+export type TaskType = {
```

Comment:
Move it to types.ts file

Reply:
Should the other types be moved

Reply:
I don't like it but as we started with horizontal architecture, we should keep to it. Create types directory in src and place file there, but name it task.ts. Only actual types that might be used in other placec should be keept there. Types for props should be kept with components